cumulusci/core/tests/test_locator_manager.py

```diff
@@ -0,0 +1,151 @@
+import unittest
```

Comment:
This module should be located in cumulusci/robotframework/tests. Our usual pattern is that if the actual code is in x/y.py, the tests for it should be in x/tests/test_y.py

Reply:
I've moved it. Thanks for catching it.